### DISCUSSION TITLE
Air-gapped Environment

### DISCUSSION THREAD

**Discussion by bwgjoseph** [podman-desktop/podman-desktop#180] [Q&A]
Hi,

Does `podman-desktop` supports installation in air-gapped environment?

And what is the minimal Windows 10 version/build requirement? Does it support Hyper-V only setup?

Thanks?

**Comment by benoitf** [comment]
Hello @bwgjoseph the Podman installer will succeed in installing it in air-gapped environment but then, when Podman machine is created it's where it might hit a fetch problem to download the VM file.

Minimal version for Windows is the one with WSL2 support

based on https://docs.microsoft.com/en-us/windows/wsl/compare-versions
WSL 2 is only available in Windows 11 or Windows 10, Version 1903, Build 18362 or later

**Comment by benoitf** [comment]
@bwgjoseph If AirGapped is a must-have for you we could enhance Podman Desktop/podman msi installer to also include the ignition file so it should work out of the box.

You may file an issue

**Reply by bwgjoseph** [reply]
Would love to have `podman-desktop` to support air-gapped environment, and if possible, also support lower Windows version (non WSL).

Will file an issue, thanks!

**Reply by rhatdan** [reply]
podman machine can pull an image from alternative sources. This means the image could be downloaded onto the air-gapped system and podman-desktop/podman-machine could take advantage of it. Not sure if this is exposed in containers.conf, but it should be. 

```
$ podman machine init --help
...
      --image-path string      Path to qcow image (default "testing")
```

@baude WDYT?
